### PR TITLE
Cousers

### PR DESCRIPTION
**Pull request by mesmerze**
implemented 
https://trello.com/c/tL6NLxiT
https://trello.com/c/RjcxQKHj
https://trello.com/c/FxlB7FSu

### CODE REVIEW

File: app/helpers/invites_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module InvitesHelper
```
Comment (kislak) [inline app/helpers/invites_helper.rb]: no need for empty helpers - just remove it

File: app/views/accounts/show.html.haml [hunk 2]
```diff
@@ -38,6 +42,18 @@
...
+	%table
+		%thead
+			%tr
+				%th Email:
+				%th Role type:
+		%tbody
+		- @roles.each do |t|
+			%tr
+				%td= t.user.email
+				%td= t.role
```
Comment (kislak) [inline app/views/accounts/show.html.haml]: use 2 spaces for indentation

File: app/assets/javascripts/invites.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (kislak) [inline app/assets/javascripts/invites.js]: remove empty files

File: app/controllers/invites_controller.rb [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  def create
+    @invite = Invite.new(user_from_id: current_user_id, user_to_id: user_to, account_id: params[:account_id])
+    if invite.valid?
+      invite.save && redirect_to(account_invites_url, notice: 'Invite have made.')
+    else
+      redirect_to account_invites_url, notice: 'Invite haven\'t been made'
+    end
+  end
+
+  def destroy
```
Comment (kislak) [inline app/controllers/invites_controller.rb]: I would put lower (after update method) - as a last method in public interface
(index, show, new, create, edit, update, delete) - I assume would be the correct order

File: app/models/invite.rb [hunk 5]
```diff
@@ -0,0 +1,24 @@
+class Invite < ApplicationRecord
+  validate :user_to_must_exits, :user_cannot_send_invites_to_himself
+  validates :user_from_id, :user_to_id, numericality: true, presence: true
+  validates :user_from_id, uniqueness: { scope: :user_to_id, message: 'two invites on one acc' }
+  belongs_to :account
+
```
Comment (kislak) [inline app/models/invite.rb]: Remove commented methods

File: spec/controllers/invites_controller_spec.rb [hunk 6]
```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+require 'pry'
```
Comment (swindler) [inline spec/controllers/invites_controller_spec.rb]: remove

File: app/controllers/invites_controller.rb [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+  def set_invite
+    @invite = Invite.find(params[:id])
+  end
+
+  def invite_params
+    params.fetch(:invite).permit(:email)
+  end
+
+  def set_user_to_id
+    @user_to = User.find_by(email: invite_params[:email]).id
```
Comment (swindler) [inline app/controllers/invites_controller.rb]: What if I enter not existing valid email? Should this be `Field should\'t be blank`?

File: app/controllers/invites_controller.rb [hunk 8]
```diff
@@ -40,9 +40,16 @@ def invite_params
...
   def set_user_to_id
-    @user_to = User.find_by(email: invite_params[:email]).id
-  rescue NoMethodError
-    redirect_to account_invites_url, notice: 'Field should\'t be blank'
+    email = invite_params[:email]
+    if email.blank?
+      redirect_to account_invites_url, notice: 'Field should\'t be blank'
+    else
+      begin
+        @user_to = User.find_by(email: email).id
```
Comment (swindler) [inline app/controllers/invites_controller.rb]: Using exceptions as system workflow control is not a good practice, actually I supposed you to check if user found and than get it's `id`

File: app/models/invite.rb (hunk 5 above)
Comment (kirylpl) [inline app/models/invite.rb]: @kislak Please check this PR updates for faster merge to master. Thank you.

**Pull request merged**